oss-vienna-happy-hour.mdx

```diff
@@ -0,0 +1,232 @@
+
+On September 16-18th, we'll be at the [Open Source Summit Europe](https://events.linuxfoundation.org/open-source-summit-europe/) in Vienna, Austria.
+
+We will have a booth in the showcase area on the ground floor where we'd love to chat with you about Postgres and Xata.
+Personally, this event is extra special for me, as it will be my first time doing booth duty as a new Xata employee.
+If you're attending the Open Source Summit, please stop by our booth and say hello!
+
+We are open source enthusiasts here at Xata, and here are a few of our projects:
+
+- [pgroll](https://github.com/xataio/pgroll): An open source PostgreSQL tool for zero-downtime, reversible schema migrations.
+- [pgstream](https://github.com/xataio/pgstream): An open source change data capture (CDC) command-line tool and library for Postgres.
+- [pgzx](https://github.com/xataio/pgzx): An open source framework to easily build PostgreSQL extensions in Zig, a systems programming language for maintainable and fast code.
+
+If you'd like to set up a time to chat about anything related to Postgres, Xata, or open source in general, feel free to email me at gulcin@xata.io.
+
+## Join us for the Postgres Happy Hour!
+
+It is not only booth duty, but I am happy to announce that we'll be also hosting a [Postgres Happy Hour at the Open Source Summit](https://events.linuxfoundation.org/open-source-summit-europe/features/co-located-events/#postgres-happy-hour-with-xata) on September 18th!
+
+At Xata, we are passionate about PostgreSQL and its role in powering modern applications.
+We believe that PostgreSQL is more than just a database; it is a fundamental technology that enables innovation and growth in the tech industry.
+And we are excited to celebrate this awesome project with the Postgres community at the Open Source Summit Europe!
+
+The event is free and open to everyone, even if you are not attending the Open Source Summit.
+However, space is limited to 50 attendees, so please fill out[this form](https://5i8caik7lja.typeform.com/psql-happy-hour) if you'd like to attend.
+
+If you're a Postgres fan and in the area, you're very welcome to join us!
+
+### Why attend?
+
+- **Engaging conversations**: Whether you’re a Postgres expert or curious about how Xata can elevate your next project, connect with like-minded individuals and explore new possibilities.
+- **Exclusive swag**: Attendees will receive some cool Xata swag as a token of appreciation.
+- **Relaxed atmosphere**: We believe the best ideas flow in a casual setting. Whether you’re diving deep into database architecture or just starting out, expect lively discussions, knowledge sharing, and maybe even a friendly debate or two.
+
+Because talking databases is always better with a drink in hand—come network, learn, and enjoy the Postgres Happy Hour!
+
+In the spirit of celebrating Postgres, we will be adhering to the community's [code of conduct](https://www.postgresql.org/about/policies/coc/).
+
+### A personal Postgres story
```

Comment:
I think this entire section was not meant to be included :)

Reply:
We want to adhere to CoC. Which part is not meant be added?